lib/spack/spack/test/sbang.py

```diff
@@ -20,6 +20,7 @@
 import spack.paths
 import spack.store
 from spack.util.executable import which
+import spack.util.spack_yaml as syaml
```

Comment:
The following will fix the style error:

```suggestion
import spack.util.spack_yaml as syaml
from spack.util.executable import which
```